README.md

```diff
@@ -56,26 +54,21 @@ This is done by adding the block below for the respective plugin to the file [we
    "path": "scaffolding",
    "repo": "hashicorp/packer-plugin-scaffolding",
    "branch": "main",
```

Comment:
I think `branch` is longer required, since we're fetching directly from the tagged release.

```suggestion
```

Reply:
Fixed thanks!